src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -29,23 +29,8 @@ async function boot(userOptions?: Partial<CircuitStartOptions>): Promise<void> {
   const jsInitializer = await fetchAndInvokeInitializers(options);
 
   const logger = new ConsoleLogger(options.logLevel);
-  Blazor.defaultReconnectionHandler = new DefaultReconnectionHandler(logger);
-
-  options.reconnectionHandler = options.reconnectionHandler || Blazor.defaultReconnectionHandler;
-  logger.log(LogLevel.Information, 'Starting up Blazor server-side application.');
-
-  const components = discoverComponents(document, 'server') as ServerComponentDescriptor[];
-  const appState = discoverPersistedState(document);
-  const circuit = new CircuitDescriptor(components, appState || '');
 
-  const initialConnection = await initializeConnection(options, logger, circuit);
-  const circuitStarted = await circuit.startCircuit(initialConnection);
-  if (!circuitStarted) {
-    logger.log(LogLevel.Error, 'Failed to start the circuit.');
-    return;
-  }
-
-  const reconnect = async (existingConnection?: HubConnection): Promise<boolean> => {
+  Blazor.reconnect = async (existingConnection?: HubConnection): Promise<boolean> => {
```

Comment:
We were setting this _after_ we tried to init the `DefaultReconnectionHandler` which needs this `reconnect` function, leading to some null refs.